daisy_workflows/image_import/debian/translate_debian.wf.json

```
@@ -20,6 +20,10 @@
     "import_subnet": {
       "Value": "",
       "Description": "SubNetwork to use for the import instance"
+    },
+    "compute_service_account": {
+      "Value": "default",
+      "Description": "Compute service account to run as"
```

```suggestion
      "Description": "Service account that will be used by the translation worker instance."
```

(I tried to use phrasing from: https://cloud.google.com/compute/docs/access/service-accounts#associating_a_service_account_to_an_instance )

---

We should use https://source.corp.google.com/piper///depot/google3/third_party/py/googlecloudsdk/surface/compute/images/export.py;l=168?q=export.py%20images&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental as a base for help text as it has been reviewed by doc writers

---

We should use the help text for --compute-service account in `gcloud beta compute images export` as a base since it was reviewed by doc writers. Can't link the source here as it's in piper.